app/templates/administrative-units.hbs

```diff
@@ -1,4 +1,5 @@
-{{breadcrumb "Besturen" route="administrative-units"}}
+{{page-title "Bestuurseenheden"}}
+{{breadcrumb "Bestuurseenheden" route="administrative-units"}}
```

Comment:
I didn't verify if this was ok, but I think it's weird to have a different breadcrumb title as the actual title on the page :smile:.

Reply:
I think it's because it's following the design (?)

Reply:
Yea, I know. I meant I didn't verify with the design team :smile:.